R/preprocessing_io.R

```diff
@@ -733,24 +733,156 @@ find_files_extract_wildcards = function(tool_results_path,
 
 #' Title
 #'
-#' @param maf_file_path
+#' @param maf_file_path path to maf that is about to be read.
+#' @param select_cols Optional parameter for specifying what columns are to be returned. If not specified, all columns will be kept.
```

Comment:
Can you describe (is it a numeric vector? Character vector of column names? Either?)

Reply:
Sorry my question was actually about `select_cols Optional parameter `

Reply:
Ah ok I see. Let's update that one as well.